packages/cli/src/cmds/beacon/handler.ts

```diff
@@ -31,6 +31,8 @@ export async function beaconHandler(args: IBeaconArgs & IGlobalArgs): Promise<vo
   const beaconPaths = getBeaconPaths(args);
   // TODO: Rename db.name to db.path or db.location
   beaconNodeOptions.set({db: {name: beaconPaths.dbDir}});
+  // we need to store to env variable so that lodestar-utils can read this
+  process.env.LODESTAR_ARCHIVE_DIR = beaconPaths.archiveDir;
```

Comment:
This is very ugly, please just pass a regular beaconNode options even if it requires a lot of plumbing to get the options to the network class